docs/configuration.md

```diff
@@ -40,10 +40,10 @@ Mandatory parameters are marked as **Required**, which means that they are requi
 
 |  Command | Description |
 |----------|-------------|
-| `max_open_trades` | **Required.** Number of trades open your bot will have. If -1 then it is ignored (i.e. potentially unlimited open trades).<br> ***Datatype:*** *Positive integer (-1 to use `"unlimited"` trades).*
+| `max_open_trades` | **Required.** Number of trades open your bot will have. If -1 then it is ignored (i.e. potentially unlimited open trades).<br> ***Datatype:*** *Positive integer or -1.*
```

Comment:
I think I have not written my idea clear enough, I'd like to keep the "unlimited" special value (imho it's better than special meaning for -1), my proposal was (refined version) "Positive integer, or -1, or `"unlimited"`. Use -1 or `"unlimited"` for unlimited number of trades." or something like that...

Reply:
Or -1 can be marked as Deprecated for this setting, to give users time to adapt to new restrictions for this value. Not in this PR in this case.

Reply:
i would do neither in this PR ... 
the current implementation does not support `"unlimited"`, but uses `-1` instead (which will be converted to `float('inf')`).

Yes, the 2 should probably be aligned.
But if you notice, i did not change any setting in this PR really - only the description (and how it's validated ...) - so doing that change in this PR seems out of place.

On the other hand -  while using -1 is ok (or "unlimited - I don't really care) - using `float('inf')` changes the type of this variable from int to either `int` or float ... (inf is no integer).

I'd like to replace `float('inf')` with maxInt (or the python equivalent ...) to allow only checking for integer, thus disallowing float values via config setting ... but until i'm certain how to approach this i'd like to leave this as is...

Reply:
ah, it does not recognize the "unlimited" string currently. That's what I missed...
Not in this PR.